src/components/EventUserTable.js

```diff
@@ -132,6 +51,42 @@ export class EventUserTable extends Component {
   }
 
   render() {
+    /**
+     * Helper function to determine whether to display action for check-in or undo check-in
+     * @param {*} rowData data about the current row
+    */
+    const changeRegistration = (event, rowData) => {
+      switch (event.target.value) {
+        case REGISTRATION_STATUS.REGISTERED:
+          if (window.confirm(`Do you want to register ${rowData.fname} ${rowData.lname}?
+This will send an email to the user.`)) {
+            this.updateUserRegistrationStatus(rowData.id, REGISTRATION_STATUS.REGISTERED);
+          }
+          break;
+        case REGISTRATION_STATUS.CHECKED_IN:
+          if (window.confirm(`Do you want to check in ${rowData.fname} ${rowData.lname}?
+This will NOT send an email to the user.`)) {
+            this.updateUserRegistrationStatus(rowData.id, REGISTRATION_STATUS.CHECKED_IN);
+          }
+          break;
+        case REGISTRATION_STATUS.WAITLISTED:
+          if (window.confirm(`Do you want to waitlist ${rowData.fname} ${rowData.lname}?
+This will send an email to the user.`)) {
```

Comment:
@ianmah what do you think of the indentation? if i indent it normally, theres a really fat indent in the pop up window

Reply:
try '\n` or some other new line character